modules/terraform/validate_struct.go

```diff
@@ -0,0 +1,99 @@
+package terraform
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/collections"
+)
+
+type ValidationOptions struct {
+	RootDir string
+	//Sub-directories relative to the RootDir to walk recursively for all Terraform modules. For example, if your
+	// RootDir is /home/project and you want to validate all modules in home/project/terraform then pass "terraform"
+	// in TargetSubDirs
+	TargetSubDirs []string
```

Comment:
Are you going for include / exclude semantics with `TargetSubDirs` and `ExcludeDirs`? If so, then you may want to name them accordingly and update the docs to match.

Reply:
Done in 2ed0bb7